src/stdatamodels/schema.py

```diff
@@ -140,7 +140,7 @@ def recurse(schema, path, combiner, ctx):
             items = schema.get('items', {})
             if isinstance(items, list):
                 for i, item in enumerate(items):
```

Comment:
```suggestion
                for item in items:
```
You don't need an `enumerate` anymore.